src/main/resources/db/migration/changes/007-Admins-create.json

```
@@ -0,0 +1,41 @@
+{ "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "Admins-1",
+        "author": "YourName",
```

Please change these as follows (you can just click "commit suggestion" to do so:

```suggestion
        "id": "007-Admins-create",
        "author": "aaasami",
```